code/modules/modular_computers/hardware/dna_processor.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/computer_hardware/dna_processor
+	name = "dna processor"
+	desc = "A dna processor used to analyze dna samples."
```

Comment:
I really hate this snowflake thing. The best solution would be making modular PC files to hold mutations and making the DNA scanner just write those to an inserted data disk.